### PR TITLE
The problem of adding the user and shelter was solved

### PR DESCRIPTION
**Pull request by fugas20**
The problem that could not add a user and a hostel was fixed, in the same way they are already shown in the tables

### CODE REVIEW

File: app/Http/Controllers/ShelterController.php [hunk 1]
```diff
@@ -44,52 +43,50 @@ public function store(Request $request)
...
-        $shelters = Shelter::find($id);
-        if ($shelters) {
-            $shelters->clearMediaCollection('logos');
-            $shelters->delete();
+        $shelter = Shelter::findOrFail($id);
+        if ($shelter) {
+            $shelter->clearMediaCollection('logos');
+            $shelter->delete();
         }
+
```
Comment (And-Y21) [inline app/Http/Controllers/ShelterController.php:76]: Hay una línea agregada

File: app/Http/Controllers/ShelterController.php [hunk 2]
```diff
@@ -115,32 +112,29 @@ public function update(Request $request, $id)
...
+        $shelter->address = $request->input('address');
+        $shelter->postal_code = $request->input('postal_code');
+
+        if ($request->hasFile('logo')) {
+            $shelter->clearMediaCollection('logos');
+            $shelter->addMediaFromRequest('logo')->toMediaCollection('logos');
         }
+        $shelter->save();
+
         return redirect()->back()->with('success', 'Albergue actualizado correctamente');
```
Comment (And-Y21) [inline app/Http/Controllers/ShelterController.php:131]: Línea agregada

File: app/Http/Controllers/ShelterController.php [hunk 3]
```diff
@@ -115,32 +112,29 @@ public function update(Request $request, $id)
...
     }
 
     public function show(Shelter $shelters)
     {
-
```
Comment (And-Y21) [inline app/Http/Controllers/ShelterController.php:143]: No habia nada?

File: app/Models/ShelterMember.php [hunk 4]
```diff
@@ -23,7 +23,7 @@ class ShelterMember extends Model implements HasMedia
     const TYPE_MEMBER = [self::TYPE_MEMBER_STAFF, self::TYPE_MEMBER_DONOR, self::TYPE_MEMBER_GODFATHER, self::TYPE_MEMBER_ADOPTER];
 
     protected $guarded = [];
-    public $timestamps = false;
+
```
Comment (And-Y21) [inline app/Models/ShelterMember.php:25]: Línea agregada

Comment (fugas20) [reply app/Models/ShelterMember.php:25]: ahorita la elimino

File: app/Http/Controllers/ShelterController.php (hunk 1 above)
Comment (fugas20) [reply app/Http/Controllers/ShelterController.php:76]: gracias ahorita la elimino

File: app/Http/Controllers/ShelterController.php (hunk 2 above)
Comment (fugas20) [reply app/Http/Controllers/ShelterController.php:131]: también ahorita la elimino

**Pull request merged**